src/components/Counter.js

```diff
@@ -47,7 +56,8 @@ class Counter extends Component {
 // parts it needs from the state object.
 const mapStateToProps = (state) => {
     return {
-        count: state.count
+        // count: state.count // Why?
```

Comment:
So count: is here is fubarbanana. But we're referencing it as count for our props because that's what it is in our store's state. then state is a variable name, also fubarbanana, but we use the variable name state because we're really referencing the state object that's coming from our store. Then we use state.count to reference the object coming in and the count property value on that state object.

Reply:
And mapStateToProps gives us an props object with the properties that we return inside the object inside mapStateToProps.